src/bp/ui-shared/src/MoreOptions/index.tsx

```diff
@@ -0,0 +1,52 @@
+import { Icon } from '@blueprintjs/core'
+import classnames from 'classnames'
+import React, { FC, Fragment } from 'react'
+
+import Button from '../Button'
+
+import style from './style.scss'
+import { MoreOptionsProps } from './typings'
+
+const MoreOptions: FC<MoreOptionsProps> = props => {
+  const { show, onToggle, items } = props
+
+  const activeClass = show && style.active
+
+  const renderLabel = item => {
+    return (
+      <Fragment>
+        {item.icon && <Icon icon={item.icon} iconSize={item.iconSize} />}
+        {item.label}
+      </Fragment>
+    )
+  }
+
+  return (
+    <Fragment>
+      <button onClick={() => onToggle(!show)} type="button" className={classnames(style.moreBtn, activeClass)}>
```

Comment:
Since this context menu could be used in a form, type="button" it is necessary especially in the context of a for where having no type would make any button without a type default to submit the form, it is good practice to always have the type of a button.